KarhooUISDK/Builders/CheckoutScreenBuilder.swift

```diff
@@ -11,22 +11,24 @@ import KarhooSDK
 
 public protocol CheckoutScreenBuilder {
     func buildCheckoutScreen(
+        navigationController: UINavigationController?,
         quote: Quote,
         journeyDetails: JourneyDetails,
         bookingMetadata: [String: Any]?,
         callback: @escaping ScreenResultCallback<KarhooCheckoutResult>
-    ) -> Screen
+    ) -> KarhooUISDKSceneCoordinator
 }
 
 public extension CheckoutScreenBuilder {
     func buildCheckoutScreen(
+        navigationController: UINavigationController?,
         quote: Quote,
         journeyDetails: JourneyDetails,
         bookingMetadata: [String: Any]? = nil,
         callback: @escaping ScreenResultCallback<KarhooCheckoutResult>
-    ) -> Screen {
-        
-        return buildCheckoutScreen(
+    ) -> KarhooUISDKSceneCoordinator {
```

Comment:
previously this method returned `Screen` as a result, so `buildCheckoutScreen` was a good name. Now it returns Coordinator, maybe you can think about better naming for this method?